DigitalExcellence/dex-frontend
# Small bugfixes

Fixed up some small bugfixes:
 - Alerts go away automatically
 - Edit contributors don't overflow
 - Project overview usernames don't overflow
 - Opening project detail modal doesn't screw up the layout
 - When entering something in the search bar and clicking projects in the navbar, the page reloads.

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1.1 Edit a project
1.2 See the alert on the top of the screen
1.3 Wait 5 seconds till the alert goes away

2.1 Edit a project
2.2 Add a contributor with a long name
2.3 See that the contributor gets hidden with 3 dots

3.1 Go to project overview
3.2 See that long usernames get hidden with 3 dots

4.1 Open a project inside project overview
4.2 Close it
4.3 See that everything is still aligned with each other


## Link to issue
#408, #400, #401, #392, #354 

Closes: #408 #400 #401 #392 #354
